client/src/App.js

```diff
@@ -1,15 +1,42 @@
+import React from 'react';
 import logo from './logo.svg';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
It would throw an ugly bug. Instead, try to add `import 'bootstrap/dist/css/bootstrap.min.css';` on the `index.js` file and all the imports would be handled.